.prettierignore

```diff
@@ -0,0 +1,6 @@
+node_modules/
```

Comment:
I added node_modules just to be specific, app/svgr because it's auto generated, and package-lock.json because it's autogenerated and it's difficult to resolve changes.